demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1486,6 +1490,14 @@ export class DemoMeetingApp
     return this.buttonStates[button] === 'on';
   }
 
+  updateButtonVideoRecordingDrop(): void {
+    if (this.buttonStates['button-record-self'] === 'on' || this.buttonStates['button-record-cloud'] === 'on'  || this.buttonStates['button-live-connector'] === 'on')
```

Comment:
nit: We have followed open and closed curly brackets for one line `if`, `else if` as well as `else`, could you please add them here?

Reply:
Sure, I will update the format